### PR TITLE
Create a branch for compute release

### PR DESCRIPTION
**Pull request by a-masterov**
## Problem
We practice a manual release flow for the compute module. This will allow automation of the compute release process.

## Summary of changes
The workflow was modified to make a compute release automatically on the branch release-compute.
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
     # It should be kept in sync with if-condition in jobs
     - cron: '0 6 * * MON' # Storage release
     - cron: '0 6 * * THU' # Proxy release
+    - cron: '0 6 * * TUE' # Proxy release
```
Comment (bayandin) [inline .github/workflows/release.yml:8]: Should be `Compute` instead of proxy?
Did they want such schedule or they want to do it on-demand and manually trigger the workflow?

Comment (a-masterov) [reply .github/workflows/release.yml]: Good catch! Fixed, thanks!

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -1123,13 +1126,15 @@ jobs:
               -f deployProxyAuthBroker=true \
               -f branch=main \
               -f dockerTag=${{needs.tag.outputs.build-tag}}
+          elif [[ "$GITHUB_REF_NAME" == "release-compute" ]]; then
+            echo Start here a workflow for the deployment of the compute release
```
Comment (bayandin) [inline .github/workflows/build_and_test.yml:1130]: We don't have a workflow for that yet, so I would rephrase it to avoid confusing anyone.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@ on:
     # It should be kept in sync with if-condition in jobs
     - cron: '0 6 * * MON' # Storage release
     - cron: '0 6 * * THU' # Proxy release
+    - cron: '0 6 * * TUE' # Compute release
```
Comment (bayandin) [inline .github/workflows/release.yml:8]: > Do they want such a schedule, or do they want to do it on-demand and manually trigger the workflow?

Any news on this one? Haven't seen Compute team doing releases on Tuesday

**Review by bayandin — APPROVED** [review]
Well done! Left a couple of suggestions and we're good to go!

Before merging we need to:
- create `release-compute` branch (from `release`)

Right after the merge:
- ask to use a `release-compute` branch instead of `release` (and a proper parameter for `release.yml` workflow)
- let people know that for compute releases they should use `release-compute-` prefixed tag (instead of `release-`)

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -49,3 +53,16 @@ jobs:
...
+
+  create-compute-release-branch:
+    if: inputs.create-compute-release-branch
+
+    permissions:
+      contents: write
+
+    uses: ./.github/workflows/_create-release-pr.yml
+    with:
+      component-name: 'Compute'
```
Comment (bayandin) [inline .github/workflows/release.yml:65]: Could you please also change `component-name: 'Storage & Compute'` above to just `component-name: 'Storage'`

**Review by bayandin — APPROVED** [review]
Looks really nice, thank you!

File: .github/workflows/release.yml (hunk 4 above)
Comment (a-masterov) [reply .github/workflows/release.yml:65]: Good catch, thanks
